src/stories/layout.stories.tsx

```diff
@@ -194,3 +195,15 @@ stories.add('Setting Panel', withState({ value: null })
         )
     )
 );
+
+stories.add('Keep aspect ratio', withInfo(
+    'A container to keep a given aspect ratio for an image or iframe'
+)(() =>
+    <div>
+        <div style={{ width: '40rem', backgroundColor: '#DDD' }} >
```

Comment:
Can we remove this div and just use the div inside the AspectRatioContainer. Also, div inside the AspectRatioContainer be so that it resizes when you make the screen smaller, i.e. so that it is responsive